index.js

```diff
@@ -15,6 +15,7 @@ function getStream(inputStream, opts) {
 
 	var p = new Promise(function (resolve, reject) {
 		stream = bufferStream(opts);
+		inputStream.on('error', reject);
```

Comment:
So wrong!, but that's the reality of Node.js streams unfortunately...

https://github.com/nodejs/readable-stream/issues/129